<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F2BF80;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #D97EB0;
border-bottom: 1px solid #BF6836;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: transparent;
</string>
         </property>
         <property name="text">
          <string>Gerador de Senhas</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #F2BF80;
border: 1px solid #BF6836;
border-radius: 2px;
</string>
        </property>
        <property name="text">
         <string>  Quantidade de Caracteres</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="label_qtd_caractere">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">alternate-background-color: #a1a1a1;</string>
        </property>
        <item>
         <property name="text">
          <string> 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 16</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="widget_4" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #BF6836;
border-radius: 2px</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_letrasMaius">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: transparent;</string>
         </property>
         <property name="text">
          <string>Letras Maiúsculas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_letrasMinus">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: transparent;</string>
         </property>
         <property name="text">
          <string>Letras Minúsculas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_num">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: transparent;</string>
         </property>
         <property name="text">
          <string>Números</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_simbolos">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: transparent;</string>
         </property>
         <property name="text">
          <string>Símbolos</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/imagem/senha-imagem.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_senha">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #F2BF80;
border: 1px solid #BF6836;
border-radius: 2px;</string>
        </property>
        <property name="text">
         <string>****************</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_gerar">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1600</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-rigth:  1px solid;
	border-rigth-color: #4f4f4f;
border-bottom: 1px solid;
	border-bottom-color: #4f4f4f;
background-color: #D97EB0;
border-radius: 2px
}

QPushButton:pressed{
border-left: 1px solid;
	border-left-color: #4f4f4f;
border-top: 1px solid;
	border-top-color: #4f4f4f;
border-right: transparent;
border-bottom: transparent;
background-color: #b4ccd0;
border-radius: 2px
}

background-color: </string>
          </property>
          <property name="text">
           <string>Gerar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #D97EB0;
border: 1px solid #BF6836;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="styleSheet">
            <string notr="true">border: transparente;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_voltar">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
border-rigth:  1px solid;
	border-rigth-color: #4f4f4f;
border-bottom: 1px solid;
	border-bottom-color: #4f4f4f;
background-color: #F2BF80;
border-radius: 2px
}

QPushButton:pressed{
border-left: 1px solid;
	border-left-color: #4f4f4f;
border-top: 1px solid;
	border-top-color: #4f4f4f;
border-right: transparent;
border-bottom: transparent;
background-color: #b4ccd0;
border-radius: 2px
}
</string>
           </property>
           <property name="text">
            <string>voltar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="maximumSize">
            <size>
             <width>0</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagens/img.qrc"/>
 </resources>
 <connections/>
</ui>
